packages/core/src/messages/packet.ts

```diff
@@ -244,7 +243,11 @@ export class Packet {
     if (isDirectMessage) {
       ticket = channel.createDummyTicket(ticketChallenge)
     } else {
-      ticket = await channel.createTicket(ticketOpts.value, ticketChallenge, ticketOpts.winProb)
+      ticket = await channel.createTicket(
+        new Balance(new BN(PRICE_PER_PACKET).mul(new BN(INVERSE_TICKET_WIN_PROB)).muln(path.length - 1)),
```

Comment:
we will be signing this ticket, so the ticket reward will not be `PRICE_PER_PACKET`, same issue as last time we discussed

Reply:
Yes, for most tickets, the reward will be **0**. But for some tickets, according to INVERSE_WIN_PROB, it will be PRICE_PER_PACKET * INVERSE_TICKET_WIN_PROB